packages/esbuild/esbuild_repo.bzl

```diff
@@ -0,0 +1,48 @@
+"""
+Helper macro for fetching esbuild versions for internal tests and examples in rules_nodejs
+"""
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+_PLATFORM_SHA = {
+    "darwin_64": "3bf980b5175df873dd84fd614d57722f3b1b9c7e74929504e26192d23075d5c3",
+    "linux_64": "9dff3f5b06fd964a1cbb6aa9ea5ebf797767f1bd2bac71e084fb0bbefeba24a3",
+    "windows_64": "826cd58553e7b6910dd22aba001cd72af34e05c9c3e9af567b5b2a6b1c9f3941",
+}
+
+_VERSION = "0.8.34"
+
+def esbuild_dependencies():
```

Comment:
Could pass in the version & a dict of platform => SHA as a param (with default values for v0.8.34) so make it easier to use. Then docs could be updated to suggest using this helper.

Reply:
From your answer above, this would introduce eager loading for users which is not great